### PR TITLE
fix(fiatconnect): fiat details screen button updates

### PR DESCRIPTION
**Pull request by satish-ravi**
### Description

- Change button text to "Submit & Continue"
- Include button as part of scroll view (this doesn't make a difference currently, but if / when there are more input fields, it scrolls correctly to the field in focus)

https://user-images.githubusercontent.com/5062591/184046220-a9769403-b3fc-4677-a6de-b4e2dfa778df.mp4

### Other changes

N/A

### Tested

Manual, unit tests

### How others should test

Navigate to fiat details screen and ensure button is "Submit & Continue"

### Related issues

- Fixes ACT-181

### Backwards compatibility

Yes

### CODE REVIEW

File: locales/base/translation.json [hunk 1]
```diff
@@ -1336,7 +1336,8 @@
     "addFiatAccountResourceExist": "You have already linked this account to Valora.",
     "addFiatAccountFailed": "Your account was not linked, please try again or contact support.",
     "selectItem": "Select an option...",
-    "selectDone": "Done"
+    "selectDone": "Done",
```
Comment (jh2oman) [inline locales/base/translation.json:1339]: do we still need this string?

**Review by jh2oman — APPROVED** [review]

File: locales/base/translation.json (hunk 1 above)
Comment (satish-ravi) [reply locales/base/translation.json:1339]: yes, this is still used in the dropdown

**Comment by ValoraQA** [comment]
Hey @MuckT, We have verified above task on Latest Android Alfajores Build V1.39.1 & updated status in test rails.

- "Submit & continue" button appears at the bottom when user navigates to the "Enter Bank Information" screen. 

**Test rail links:**
**Android 11:** https://valoraapp.testrail.io/index.php?/tests/view/61285

**Devices:** Samsung Galaxy F41(11.0)
CC: @satish-ravi

Thanks..!!

**Comment by ValoraQA** [comment]
Hey @MuckT, We have verified above task on Latest iOS Alfajores Build V1.39.1 & updated status in test rails.

**Observation** :

- "Submit & continue" button appears at the bottom when user navigates to the "Enter Bank Information" screen.

**Test rail links:**
**iOS 15:** https://valoraapp.testrail.io/index.php?/tests/view/61841

**Device**: iPhone 13 mini(15.1.1)

**CC**: @satish-ravi

Thanks..!!

**Pull request merged**